intuita-webview/src/sourceControl/CommitView/index.tsx

```diff
@@ -110,51 +123,50 @@ const CreatePR = ({
 						</VSCodeDropdown>
 					</div>
 				) : null}
-				<div className={styles.formField}>
-					<label htmlFor="targetBranch">Target branch:</label>
-					<VSCodeDropdown
-						id="targetBranch"
-						value={targetBranch}
-						onChange={onChangeFormField('targetBranch')}
-					>
-						{targetBranchOptions.map((opt, index) => (
-							<VSCodeOption value={opt} key={index}>
-								{opt}
-							</VSCodeOption>
-						))}
-					</VSCodeDropdown>
-				</div>
-				<div className={styles.formField}>
-					<label htmlFor="baseBranch">Base branch:</label>
-					<VSCodeDropdown
-						id="baseBranch"
-						value={baseBranch}
-						onChange={onChangeFormField('baseBranch')}
+				<CommitForm
+					formData={formData}
+					onChangeFormField={onChangeFormField}
+				/>
+				{/* @TODO should we even allow to unapply all jobs? */}
```

Comment:
We could, but also we might get rid of this feature because Alex said it made little sense to him.